.github/workflows/publish-solidity-sdk.yml

```diff
@@ -17,25 +17,31 @@ jobs:
           node-version-file: "package.json"
           registry-url: "https://registry.npmjs.org"
 
```

Comment:
Unless I'm misunderstanding, if you follow my suggestions above, you can get rid of this workflow entirely.

Reply:
We aim to to have different tags and different workflows for publishing separate packages. Also we need to publish only solidity sdk and js sdk. But there are bunch of other packages here as well. How to achieve this?

Reply:
This makes more sense!

Eventually I'll move the monorepo to a proper release management tool, e.g. I've used https://github.com/changesets/changesets in the past and I find it quite nice and there are a few other similar things out there.  For now I don't have any better suggestions than what you're doing if you want to release packages independently.